.gitignore

```diff
@@ -10,6 +10,7 @@ node_modules
 
 # production
 /build
+.vscode
 
```

Comment:
- [ ] Although you've added `.vscode` to `.gitignore`, it is still visible on github. You need to manually remove it. Please, let's remove the file from the repo as mentioned in the previous reviews. :pray: